nilearn/datasets/neurovault.py

```diff
@@ -44,6 +45,11 @@
 _DEFAULT_BATCH_SIZE = 100
 _DEFAULT_MAX_IMAGES = 100
 
+AFFINE_STD = np.array([[3., 0., 0., -90.],
```

Comment:
2 things:
* I think that the standard affine you propose corresponds to a certain standard shape (e.g. to have approximate symmetry in x). I would add it.
* What if the use wants to resample differntly, e.g. in 2- or 4-mm resolution ? Could he/she provide an affine ?